packages/bull-vault/src/AuctionBull.sol

```diff
@@ -398,6 +398,14 @@ contract AuctionBull is UniFlash, Ownable, EIP712 {
         return _getCurrentDeltaAndCollatRatio();
     }
 
+    /**
+     * @notice allows an order to be cancelled by marking its nonce used for a given msg.sender
+     * @param _nonce the nonce to mark as used
+     */
```

Comment:
for users to cancel orders manually?

Reply:
yep exactly, we have it in crabv2 as well